source/deployment/sso-saml-adfs-msws2016.rst

```diff
@@ -0,0 +1,208 @@
+Configure SAML with Microsoft ADFS using Microsoft Windows Server 2016
+======================================================================
+
+The following process provides steps to configure SAML 2.0 with Microsoft ADFS for Mattermost and Microsoft Windows Server 2016.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+.. include:: sso-saml-before-you-begin.rst
+
+The following are basic requirements to use ADFS for Mattermost:
+ - An Active Directory instance where all users have a specified email and username attributes. For Mattermost servers running 3.3 and earlier, users must also have their first name and last name attributes specified.
+ - A Microsoft Server running. The screenshots used in this guide are from Microsoft Server 2012R2, but similar steps should work for other versions.
+ - An SSL certificate to sign your ADFS login page.
+ - ADFS installed on your Microsoft Server. You can find a detailed guide for deploying and configuring ADFS in `this article <https://msdn.microsoft.com/en-us/library/gg188612.aspx>`__.
+
+On your ADFS installation, note down the value of the **SAML 2.0/W-Federation URL** in ADFS Endpoints section, also known as the **SAML SSO URL Endpoint** in this guide. If you chose the defaults for the installation, this will be ``/adfs/ls/``.
```

Comment:
Would it be possible to include the path to the ADFS Endpoints section? So, something like: 
```suggestion
On your ADFS installation, navigate to "Menu name" > ADFS Endpoints and note down the value of the **SAML 2.0/W-Federation URL**/**SAML SSO URL Endpoint**. If you chose the defaults for the installation, this will be ``/adfs/ls/``.
```